packages/ensnode-sdk/src/shared/block-ref.ts

```diff
@@ -2,32 +2,24 @@ import type { BlockRef } from "./types";
 
 /**
  * Compare two {@link BlockRef} object to check
- * if one is before or same as the other.
+ * if one is before the other.
```

Comment:
```suggestion
 * if blockA is before blockB.
```

Goal: Be specific about relationship between params.